src/undo/undo_manager.cpp

```diff
@@ -41,7 +41,7 @@
 
 namespace OpenOrienteering {
 
-Q_STATIC_ASSERT(UndoManager::max_undo_steps < std::numeric_limits<int>::max());
+Q_STATIC_ASSERT(static_cast<int>(UndoManager::max_undo_steps) < std::numeric_limits<int>::max());
```

Comment:
max_undo_steps is of `std::size_t` which could be to big for `int`.
But `std::numeric_limits<int>::max()` is never to big for `unsigned int`.